device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -113,7 +113,9 @@
 
     private final boolean isMultiplexing;
 
-    public AmqpsIotHubConnection(DeviceClientConfig config, String transportUniqueIdentifier, boolean isMultiplexing)
```

Comment:
did you remove the parameter because it can be inferred?

Reply:
Yes. ```isMultiplexing``` was only ever one value